cpu/atmega_common/Makefile.include

```diff
@@ -1,6 +1,15 @@
 # Target architecture for the build. Use avr if you are unsure.
 export TARGET_ARCH ?= avr
 
+# define build specific options
+CFLAGS_LINK  = -ffunction-sections -fdata-sections
+
+# export compiler flags
+export CFLAGS += $(CFLAGS_LINK)
+
+# export linker flags
+export LINKFLAGS += -Wl,--gc-sections -static -lgcc
```

Comment:
Just copy/paste from other Makefile.include, I'm not sure if that's needed.. 
I can't even find documentation of the -lgcc option ....